wordpress-mobile/react-native-libraries-publisher
# Add `react-native-reanimated` library with version `2.17.0`

Adds the Reanimated library. In newer versions, we no longer need [the forked repository](https://github.com/wordpress-mobile/react-native-reanimated) because we can now set the Hermes engine using an environment variable. Previously, we had to enforce this by updating the build configuration ([reference](https://github.com/wordpress-mobile/react-native-reanimated/pull/3/files#diff-49c2df30272bc7a6770fc503b2304079ad0a171dddc7b9015c354acccabdc17eR13)).

**NOTE:** This version only works in newer versions of React Native, hence it was branched off from https://github.com/wordpress-mobile/react-native-libraries-publisher/pull/23.

## How to test
1. Run the command `npm install`.
2. Run the command `CLIENT_SIDE_BUILD="True" JS_RUNTIME="hermes" ./gradlew react-native-reanimated:publishToMavenLocal -exclude-task prepareToPublishToS3`
3. Observe that the library is built and published successfully in the local folder `~/.m2/repository/org/wordpress-mobile/react-native-libraries`.
